### PR TITLE
Feat/pex proof request flow

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

**Review by SamHellawell — APPROVED** [review]
looks good for this update, but just checking in general if legacy QR code requests are still supported? we will phase them out soon but not immedtiately due to people using them

File: src/features/qr-code-scanner/qr-code.js [hunk 1]
```diff
@@ -312,7 +313,33 @@ export async function relayServiceHandler(json) {
   }
 }
 
+export async function pexMessageHandler(value) {
```
Comment (mike-parkhill) [inline src/features/qr-code-scanner/qr-code.js:316]: this file's starting to get big - should we consider starting to break the handlers out into their own files?

Comment (maycon-mello) [reply src/features/qr-code-scanner/qr-code.js:316]: yes, I think we can also improve performance. Each handler perform a JSON.parse, we can do it once and pass along for all the handlers, same for fetching data from URL. Let me create a jira to perform this properly

Comment (SamHellawell) [reply src/features/qr-code-scanner/qr-code.js:316]: was thinking the same but would argue its better to refactor in later PR (unless its super quick to do) - just want to land this so i can finish up the certs side and have the team test it too

Comment (maycon-mello) [reply src/features/qr-code-scanner/qr-code.js:316]: https://dock-team.atlassian.net/browse/DCKA-1251

**Comment by maycon-mello** [comment]
> looks good for this update, but just checking in general if legacy QR code requests are still supported? we will phase them out soon but not immedtiately due to people using them

Yes, legacy flow is still working

File: src/features/qr-code-scanner/qr-code.js (hunk 1 above)
Comment (mike-parkhill) [reply src/features/qr-code-scanner/qr-code.js:316]: Yep, I'm good with doing it later, just wanted to call it out

**Pull request merged**